warp/src/raygun/mod.rs

```diff
@@ -29,6 +31,8 @@ pub enum RayGunEventKind {
     ConversationArchived { conversation_id: Uuid },
     ConversationUnarchived { conversation_id: Uuid },
     ConversationDeleted { conversation_id: Uuid },
+
+    CommunityCreated { community_id: Uuid },
```

Comment:
Thoughts about adding a flag to `ConversationCreated` to indicate the type of conversation it is?

Reply:
Yeah it could come in handy if in the ui we need screens where we show only DirectConversations or only GroupConversations. That way we wouldn't need to fetch a conversation if we weren't interested in that ConversationType. (At the moment though, they are both listed together on the same list, like discord does, so they all need to be fetched anyway.) Alternatively it could be split into different events, which would achieve the same effect, perhaps even better since the ui would just directly subscribe to the events they need instead of checking the conversation type on each event emitted. Either way, probably best to do that in a separate PR from this one.